extensions/extensions-helm/extensions-helm-pipeline-invocation/aissemble-pipeline-invocation-app-chart/README.md

```diff
@@ -10,16 +10,18 @@ helm install pipeline-invocation-service oci://ghcr.io/boozallen/aissemble-pipel
-| ingress.spec.rules.hosts                                                                             | A list of hosts for ingress to support, each with their own path definition                                                              | No                |                                                     |
-| ingress.status                                                                                       | Load balancer IP if required                                                                                                             | No                | None                                                |
+| Property                                                                                             | Description                                                                                                      | Required Override | Default                                             |
+|------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------|-------------------|-----------------------------------------------------|
+| ingress.apiVersion                                                                                   | k8s API version to use                                                                                           | No                | networking.k8s.io/v1                                |
+| ingress.enabled                                                                                      | k8s Whether to enable ingress                                                                                    | No                | false                                               |
+| ingress.kind                                                                                         | Type of kubernetes entity                                                                                        | No                | Ingress                                             |
+| ingress.metadata.name                                                                                | Name of the ingress                                                                                              | No                | pipeline-invocation-service-web                     |
+| ingress.metadata.annotations.kubernetes.io/ingress.class                                             | Ingress class name                                                                                               | No                | nginx                                               |
+| ingress.metadata.annotations.ingress.metadata.annotations.nginx.ingress.kubernetes.io/server-snippet | Custom configurations for the nginx ingress class                                                                | No                | gunzip on; gzip on; gzip_proxied any; gzip_types *; |
```

Comment:
A: I know you didn't add this, but this path looks incorrect. Why is it duplicating `ingress.metadata.annotations`?